webapp/src/app.tsx

```diff
@@ -5614,6 +5614,13 @@ document.addEventListener("DOMContentLoaded", async () => {
     }
 
     await auth.initAsync();
+
+    // Handle logins that start via query parameter (for now, this is only Clever)
+    const autoLogin = query["autologin"] as pxt.IdentityProviderId;
+    if (autoLogin) {
+        await auth.loginAsync(autoLogin, true);
```

Comment:
Note: The `auth.loginAsync` code path eventually checks if the provided identity provider is a valid one, so we don't need to check it here.